leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisRegistrationStore.java

```diff
@@ -942,6 +948,24 @@ public Builder setRegistrationSerDes(RegistrationSerDes registrationSerDes) {
             return this;
         }
 
+        /**
+         * Set {@link LwM2mIdentitySerDes} instance used to LwM2mIdentity serialize/de-serialize {@link LwM2mIdentity}
+         * to/from this store.
+         */
+        public Builder setIdentitySerDes(LwM2mIdentitySerDes identitySerDes) {
+            this.identitySerDes = identitySerDes;
+            return this;
+        }
+
+        /**
+         * Set {@link LwM2mPeerSerDes} instance used to LwM2mIdentity serialize/de-serialize {@link LwM2mPeer} to/from
```

Comment:
> Set {@link LwM2mIdentitySerDes} instance used to ~LwM2mIdentity~ serialize/de-serialize {@link LwM2mIdentity}

Maybe `LwM2mIdentity` is not intended  too ?

Reply:
This is also corrected